assets/xml/objects/object_fish.xml

```diff
@@ -1,162 +1,315 @@
 ﻿<Root>
+    <!-- Assets for fishing ?? -->
```

Comment:
I feel you could drop the question marks here; it's pretty clear what these assets are for now that you're done with the file.

Reply:
nah yea it was more cus like iirc mm doesnt actually have the fishing minigame?? idk but yea ill remove

Reply:
Yeah, it's a bit odd. MM has the fishing actor, but no fishing minigame, and I'm not sure that spawning the fishing actor would even work in practice. Still, it's pretty clear what these assets were intended to be used for.